pom.xml

```diff
@@ -125,7 +125,7 @@
     <hadoop.version>2.7.4</hadoop.version>
     <protobuf.version>2.5.0</protobuf.version>
     <yarn.version>${hadoop.version}</yarn.version>
-    <zookeeper.version>3.4.6</zookeeper.version>
+    <zookeeper.version>3.4.14</zookeeper.version>
```

Comment:
Is it safe to upgrade `zookeeper` to `3.4.14` for Hadoop 2.7?

Reply:
Yes that's a good question @beliefer - what does 2.7 use?
I suspect we can update it. In that case we don't need to special-case the ZK version below anymore.

Reply:
Hadoop 2.7 uses `zookeeper:3.4.6` by default.
It's OK. I reference https://www.cnblogs.com/ywjfx/p/11344345.html.

Reply:
Could you try to upgrade zookeeper to 3.4.14 for [Hadoop 2.7.4](https://github.com/apache/hadoop/tree/release-2.7.4-RC0), and check if Hadoop test can pass?